Sources/Core/EthereumNetwork/RequestParameter/APIRequestParameterType.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Web3+APIRequestParameter.swift
+//  Web3swift
```

Comment:
Please. update `Web3+APIRequestParameter.swift` to `APIRequestParameter.swift` and `Web3swift` to be lowercase.